phase2/kubeadm/configure-vm-kubeadm-node.sh

```diff
@@ -1,4 +1,4 @@
 
 # This is not meant to run on its own, but extends phase2/kubeadm/configure-vm-kubeadm.sh
 
-kubeadm join --token "$KUBEADM_TOKEN" "$KUBEADM_MASTER_IP:443" --skip-preflight-checks --discovery-token-unsafe-skip-ca-verification true
```

Comment:
:+1:
also, we should pass this flag ONLY if the kubeadm version is v1.8 or higher.
The versions below that didn't have this feature.

Reply:
Agreed. :+1: 

@shashidharatd I'm not familiar with kubeadm version management/check in kubernetes-anywhere. Could you please help fix this? Thanks!

Reply:
Maybe can ues `KUBEADM_VERSION`, already have this variable in this file.
But `KUBEADM_VERSION` is a link like :`gs://kubernetes-release-dev/bazel/v1.10.0-alpha.0.631+e7ad6e60081887/bin/linux/amd64/`.

Reply:
yes but the variable is also used in openstack and seems like there it uses "stable" version from another link. It makes it complicated to check the version. And I can't test it out on an gce cloud :(

Reply:
yep, its complicated. off hand what i could think of is to extract a `semver` from KUBEADM_VERSION and write a `semver` matching function and based on the condition use the right option for `--discovery-token-unsafe-skip-ca-verification` flag.

AFAIK, the `KUBEADM_VERSION` can be either one of 2 values.
- `stable`
- an URL pointing to GCS like the one prefixed with `gs://`

Reply:
@shashidharatd sounds great! Thanks! BTW, do you have cycle to help implement this?

Reply:
@xiangpengzhao, i am giving it a try. will raise a PR once my GCE local testing pass.

Reply:
@shashidharatd cool, thanks!